tests/unit/convert_gs1_xml_to_json.t

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/perl -w
+
+use Modern::Perl '2017';
+use utf8;
+
+use JSON;
+
+use Test::More;
+use Test::Number::Delta relative => 1.001;
+use Log::Any::Adapter 'TAP';
+
+use ProductOpener::GS1 qw/:all/;
+use ProductOpener::Test qw/:all/;
+
+use JSON "decode_json";
+
+my ($test_id, $test_dir, $expected_result_dir, $update_expected_results) = (init_expected_results(__FILE__));
+
+my $input_dir = "$test_dir/inputs/$test_id";
+
+opendir(my $dh, $input_dir) or die("Could not open the $input_dir directory: $!\n");
+
+foreach my $file (sort(readdir($dh))) {
+
+	next if $file !~ /\.xml$/;
+	my $file = $`;    # remove xml extension
+	my $json_file = "$input_dir/$file.json";
```

Comment:
You should instead use a File::Temp->newdir(); and put you files there (see usage in other tests)

Reply:
ok.